garden_ai/client.py

```diff
@@ -362,6 +364,114 @@ def register_pipeline(self, pipeline: Pipeline, container_uuid: str) -> str:
         local_data.put_local_pipeline(registered)
         return func_uuid
 
+    def _create_minimal_step(
```

Comment:
Can you add dcostring comments to these new methods? And maybe even add a sort of section delimiter to show to someone scrolling through that they're related?